iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/processing/oozie_app/workflow.xml

```diff
@@ -276,6 +276,11 @@
             <value>${sparkExecutorMemory}</value>
             <description>memory for individual executor</description>
         </property>
+        <property>
+            <name>citationmatchingDirectSparkExecutorOverhead</name>
+            <value>2048</value>
```

Comment:
I don't understand why this value has to be specified both here and in the `config-default.template`. And to they have to be always kept in sync?

Reply:
This is in line with other `*SparkExecutorOverhead` properties defined in `workflow.xml` file and it does not require keeping in sync values declared in `workflow.xml` file with the ones from the `default-config.xml` files. 

I think this duplication was mostly kept as safety measure: if the value is not declared in `default-config.xml` it will be taken from the `workflow.xml` as a default one. It is quite useful when performing development runs (no `default-config.xml` involved) but I agree it does not look well having some properties defined in two places therefore I have created a dedicated issue to consider duplicates removal: #1430.